code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -37,12 +37,12 @@
 /mob/living/simple_animal/hostile/syndicate/Die()
 	..()
 	if(corpse)
-		new corpse (src.loc)
+		corpse.createCorpse()

```

Comment:
When you have a variable of a type like this it should read the data from the type path of the variable, I believe I've made something almost identical to this work with windows in the past which is why I've applied it here. 

But I'll be certain whether or not it works soon.